### DISCUSSION TITLE
Stoplight Prism- Running on the devops pipeline using cli

### DISCUSSION THREAD

**Discussion by vj-techlabs** [stoplightio/prism#1847] [Q&A]
I am working on running Prism using cli on the Devops pipeline for unit testing.

Problem:
Using Cli I am able to start the mock server and once it is started the control never comes out of the step, I understand that is because of the autoreloading feature in Prism. It keeps on looking at the open api document for any changes. 
Because of this the CI pipeline never finishes the prism step and keep on running. As a result I have to manually kill the job in pipeline.

Is there anyway in Cli to start the mock and stop the mock server using the commands.
Eg:
prism mock .... start
prism mock ..... stop

**Comment by vinz178** [comment]
Same issue here.

**Comment by ponkyky** [comment]
You can launch a process in background by appending  " &" at the end of the command:  
`prism mock .... &`

The output would still be active in the current shell, if you want to remove output as well you can redirect it to /dev/null, and at the same time redirect error output to standard output with "2&>1" if needed:
 `prism mock ....  > /dev/null 2&> 1 &`
 
 Because the job is not in the foreground it shouldn't be a problem with jobs running forever.  
 If this is the case you can kill the background job with "kill" and the id number of backgrounded job, he should be the first but you can have a list of running background jobs with the command "jobs" if needed :
 `kill %1`
